src/base/bittorrent/torrent.h

```diff
@@ -193,7 +193,8 @@ namespace BitTorrent
         virtual qreal ratioLimit() const = 0;
         virtual int seedingTimeLimit() const = 0;
 
-        virtual QStringList absoluteFilePaths() const = 0;
+        virtual QVector<QString> files() const = 0;
```

Comment:
I think the name is too obscure, I like `filePaths` better.

Reply:
>I think the name is too obscure, I like `filePaths` better.

Ok. Will revert it back. But...
Currently I feel this method to be a bit redundant. It does not do anything special that could not be done using the current basic methods (`filesCount()` and `filePath()`).
I'm considering the idea of having an internal cached list of files (filtered and stored as QStrings) instead of getting/converting file names from the underlying libtorrent classes. Then this method would make more sense, being an ordinary getter. What do you think?

Reply:
>Then this method would make more sense, being an ordinary getter. What do you think?

The idea is good. But you will need to watch out that sometimes it really needs to get up-to-date list from libtorrent classes, for example when file rename operation occurs libtorrent might replace some invalid character IIRC.